legal-api/src/legal_api/models/director.py

```diff
@@ -61,6 +63,14 @@ def json(self):
             if 'addressType' in director_address:
                 del director_address['addressType']
             d['deliveryAddress'] = director_address
+        if self.mailing_address:
+            director_mailing_address = self.mailing_address.json
+            if 'addressTYpe' in director_mailing_address:
```

Comment:
Should this be addressTYpe or addressType

Reply:
good catch!

Reply:
I was planning to add tests in a separate pull request in the interest of keeping pull requests small. I do have tests surrounding this that go along with the actual change of director filing work